### DISCUSSION TITLE
Is there a way to customize the Json Serializer used by GraphQL?

### DISCUSSION THREAD

**Discussion by ghost** [graphql-java/graphql-java#2541] [Q&A] [answered]
For example, don't include properties with "null" value in the response.

**Comment by bbakerman** [comment, marked as answer]
The base graphql-java library does not do ANY json serialisation.

I suspect you are using a wrapping framework that is doing that for you.

**Reply by ghost** [reply]
Thanks. Ooops, sorry.I may be in the wrong project then. Yeah, we are using the new spring-graphl (w/spring boot)
